dazel.py

```diff
@@ -6,7 +6,7 @@
 import shutil
 import subprocess
 import sys
-import types
```

Comment:
I'm curious - how did it ever work?! Weird we didn't catch it until now...

Anyways, can you please extract the change with types/collections to a separate pull request? This will make the envs change much simpler.

In the seperate PR feel free to not even change the version in the setup.py. Just change types to collections, and send me a review.

Thanks.

Reply:
I have no idea haha. Sure will make a new PR.